engine/config_integrations/manual.py

```diff
@@ -12,39 +12,28 @@
 description = None
 
 # Default templates
-slack_title = """{% set metadata = payload.view.private_metadata %}
-{%-if "message" in metadata -%}
-{% set title = "Message from @" + metadata.author_username %}
-{%- else -%}
-{% set title = payload.view.state["values"].TITLE_INPUT.FinishCreateIncidentViewStep.value %}
-{%- endif -%}
-*<{{ grafana_oncall_link }}|#{{ grafana_oncall_incident_id }} {{ title }}>* via {{ integration_name }}
+slack_title = """{{ payload.oncall.title }}
+ *<{{ grafana_oncall_link }}|#{{ grafana_oncall_incident_id }} {{ title }}>* via {{ integration_name }}
```

Comment:
I think it's better to start title from incident_id, like it was before. BTW what is {{ title }} here?
```suggestion
slack_title = """*<{{ grafana_oncall_link }}|#{{ grafana_oncall_incident_id }} {{ payload.oncall.title }}>*
 via {{ integration_name }}
```